### PR TITLE
added upload file component with tests, get and put file functions

### PR DESCRIPTION
**Pull request by letrylesha**
image

### CODE REVIEW

File: src/components/FileUpload/FileUploadStyles.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  position: relative;
+`;
+
+export const UploadFileInputContainer = styled.input`
+  font-size: 0;
+  width: 200px;
+  height: 30px;
+  display: block;
+  cursor: pointer;
+  border: none !important;
```
Comment (drews-believer) [inline src/components/FileUpload/FileUploadStyles.ts:40]: important?)

File: src/components/App/Auth/Auth.tsx [hunk 2]
```diff
@@ -41,10 +41,19 @@ const AuthSideH3 = styled.h3`
...
   justify-content: center;
   align-items: center;
+  overflow-y: auto;
+
+  &::-webkit-scrollbar { width: 3px; height: 3px;}
+  &::-webkit-scrollbar-button {  background-color: #fff; }
+  &::-webkit-scrollbar-track {  background-color: #fff;}
+  &::-webkit-scrollbar-track-piece { background-color: #ffffff;}
+  &::-webkit-scrollbar-thumb { height: 50px; background-color: rgba(36,195,249, 1); border-radius: 10px;}
+
```
Comment (didevgen) [inline src/components/App/Auth/Auth.tsx:56]: Don't do this

File: src/components/App/Auth/Signup/Signup.tsx [hunk 3]
```diff
@@ -108,4 +119,8 @@ const Signup: React.FC<Props> = props => {
   );
 };
 
-export default connect(null, { authSignup })(Signup);
+const mapStateToProps = (state: any) => ({
+  uid: state.authReducer.uid
```
Comment (didevgen) [inline src/components/App/Auth/Signup/Signup.tsx:123]: uuid or guid

**Review by didevgen — CHANGES_REQUESTED** [review]

**Review by drews-believer — APPROVED** [review]

File: src/components/FileUpload/FileUploadStyles.ts (hunk 1 above)
Comment (letrylesha) [reply src/components/FileUpload/FileUploadStyles.ts:40]: ыыыы

**Pull request closed without merging**